spec/pluto/operation/horizontal_blur_spec.cr

```diff
@@ -17,6 +17,13 @@ describe Pluto::Operation::HorizontalBlur do
       expect_digest image, "d7fa6faf6eec5350f8de8b41f478bf7e8d217fa9"
       expect_digest blurred_image, "625be82cf07186fde56a81059c8149bc192bb1c9"
     end
+
+    it "doesn't cause arithmetic overload" do
+      with_sample("problem_images/28_arithmetic_overflow_in_blur.jpg") do |io|
```

Comment:
Using new image from https://github.com/phenopolis/pluto-samples/pull/1 to verify fix.

Reply:
What license does the new image have?

Reply:
I grabbed the moon image from unsplash.com, which I assumed is where the original pluto image came from (discovered the Unsplash license reference in pluto-samples shard.yml file).